libcxx/include/fstream

```diff
@@ -362,6 +362,9 @@ private:
   bool __read_mode();
   void __write_mode();
 
+  _LIBCPP_HIDE_FROM_ABI static int __fseek(FILE* __file, pos_type __offset, int __whence);
+  _LIBCPP_HIDE_FROM_ABI static pos_type __ftell(FILE* __file);
```

Comment:
I think we can just make these free functions. (I'd just define them at the top of the file). It would also be good to remove the `!defined(_LIBCPP_MSVCRT)` from the definition of `_LIBCPP_HAS_OFF_T_FUNCTIONS` and probably just inline `_LIBCPP_HAS_OFF_T_FUNCTIONS` as a whole, since at that point it's just equivalent do `defind(_NEWLIB_VERSION)`.

Reply:
I kept the functions in the class, as they use `pos_type`, but I updated the PR to remove the unnecessary intermediate define `_LIBCPP_HAS_OFF_T_FUNCTIONS`.